docs/DeveloperGuide.md

```diff
@@ -295,30 +295,146 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `EduTrack` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+---
+
+**Use case: Remove a class**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to view the list of classes
+2.  EduTrack shows a list of classes
+3.  User requests to delete a specific class in the list
+4.  EduTrack request for confirmation and waits for y/n response from user
```

Comment:
I like how clear and concise your MSS is. I think there is a small grammar mistake for "request", might be "requests" instead 😄

Reply:
Thanks for spotting this mistake 👍 Its been fixed